index.html

```diff
@@ -12,84 +12,76 @@
+<li>This document provides guidance specifically for mobile applications on phones and tablets; it does not include guidance for other types of mobile devices such as wearables and laptops.</li>
 <li>This document does not propose changes to WCAG 2 or its supporting documents; it does not include interpretations for implementing WCAG 2 in web technologies.</li>
-<li>This document is not sufficient by itself to ensure accessibility in Mobile Applications. As a web standard, WCAG does not fully cover all accessibility requirements for non-user interface aspects of platforms, user-interface components as individual items, nor closed product software (where there is no Assistive Technology to communicate programmatic information).</li>
+<li>This document is not sufficient by itself to ensure accessibility in mobile applications &mdash; as a web standard, WCAG does not fully cover all accessibility requirements for non-user interface aspects of platforms, user-interface components as individual items, or closed product software (where there is no assistive technology to communicate programmatic information).</li>
 <li>This document does not comment on hardware aspects of products, because the basic constructs on which WCAG 2 is built do not apply to these.</li>
-<li>This document does not comment on WCAG Level AAA success criterion.</li>
-<li>This document does not provide supporting techniques for implementing WCAG 2 in Mobile Applications.</li>
-<li>This document is purely an informative note about Mobile Applications, not a standard, so it does not describe how Mobile Applications should conform to it.</li>
+<li>This document does not comment on WCAG Level AAA success criteria.</li>
+<li>This document does not provide supporting techniques for implementing WCAG 2 in mobile applications.</li>
+<li>This document is purely an informative note about mobile applications, not a standard, so it does not describe how mobile applications should conform to it.</li>
 </ul>
 
-    <h3>Document Overview</h3>
-<p>This document includes text quoted from the WCAG 2.2 principles, guidelines, success criteria, and glossary definition  without any changes. This document also includes text quoted from the WCAG2ICT document. The guidance provided by this document for each principle, guideline, success criterion, and definition is preceded by a heading beginning with “Applying…”. This guidance was created by the Mobile Accessibility Task Force.</p>
+    <h3>Document overview</h3>
+<p>This document includes text quoted from the WCAG 2.2 principles, guidelines, success criteria, and glossary definitions  without any changes. This document also includes text quoted from the WCAG2ICT document. The guidance provided by this document for each principle, guideline, success criterion, and definition is preceded by a heading beginning with “Applying…”.</p>
 
-    <h3>Document Conventions</h3>
+    <h3>Document conventions</h3>
```

Comment:
WCAG2ICT has capital 'C' for Conventions

Reply:
Slowly shifting to sentence case rather than using a pascal case type. Didn't get to WCAG2ICT before it was published